### PR TITLE
Notification basic crud

### PR DESCRIPTION
**Pull request by havriutkin**

### CODE REVIEW

File: src/main/java/com/shinobicoders/teamcodeapi/service/NotificationService.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    public Notification createNotification(Notification notification) {
+        return notificationRepository.save(notification);
+    }
+
+    public Notification updateNotification(Long id, Notification notificationDetails) {
+        Notification notification = notificationRepository.findById(id)
+                .orElseThrow(() -> new RuntimeException("Notification not found with id: " + id));
+
+        notification.setId(notificationDetails.getId());
```
Comment (OleksiiPliekhov) [inline src/main/java/com/shinobicoders/teamcodeapi/service/NotificationService.java]: The same situation as one in UserService

File: src/main/java/com/shinobicoders/teamcodeapi/controller/NotificationController.java
Comment (OleksiiPliekhov) [inline src/main/java/com/shinobicoders/teamcodeapi/controller/NotificationController.java:1]: Add cases to edit and delete notifications

Comment (havriutkin) [reply src/main/java/com/shinobicoders/teamcodeapi/controller/NotificationController.java:1]: Currently we don't want users to be able to do that. Thus, we have delete and update in NotificationService for internal use, but not in NotificationController

File: src/main/java/com/shinobicoders/teamcodeapi/service/NotificationService.java (hunk 1 above)
Comment (havriutkin) [reply src/main/java/com/shinobicoders/teamcodeapi/service/NotificationService.java]: Fixed

**Review by OleksiiPliekhov — APPROVED** [review]

**Pull request merged**